package.json

```diff
@@ -12,6 +12,7 @@
     "test": "jest --runInBand",
     "build": "rm -rf dist/ && tsc",
     "lint": "tslint --project . -t verbose",
+    "lint:tests": "tslint -c tslint.json '__tests__/**/*.test.ts'",
```

Comment:
Can we just also lint the tests as a part of "lint", or is there a reason it's separate?